multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/ndarray/complex/_NDArray.kt

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.kotlinx.multik.ndarray.complex
+
+import org.jetbrains.kotlinx.multik.ndarray.data.Dimension
+import org.jetbrains.kotlinx.multik.ndarray.data.NDArray
+import org.jetbrains.kotlinx.multik.ndarray.operations.map
+
+/**
+ * Transforms this [NDArray] of [ComplexFloat] to an [NDArray] of the real part of complex numbers.
+ * Dimensions are preserved.
+ *
+ * @param D dimension.
+ * @return [NDArray] of real portion of [ComplexFloat]
+ */
+@get:JvmName("reFloat")
+public val <D : Dimension> NDArray<ComplexFloat, D>.re: NDArray<Float, D>
```

Comment:
Please make the extension from `MultiArray`, since we are creating a new array, not changing the current one.